setup.py

```diff
@@ -0,0 +1,17 @@
+"setup.py"
+
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+extensions = [
+    Extension("soletta", ["bindings/python/mainloop.pyx"],
+        include_dirs = ["./build/soletta_sysroot/usr/include/soletta"],
```

Comment:
see my overall comment with a suggestion for your prefix problem. You can pass it using the command line (get with `sys.argv` or argparse) or environment variable (get with `os.environ`). then you define those on Makefile before calling this script.